templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -61,6 +61,17 @@ contents:
       fi
     }
 
+    #backup latest static pod resources for kube-apiserver
+    backup_latest_kube_static_resources() {
+      echo "Trying to backup latest static pod resources.."
+      LATEST_STATIC_POD_DIR=$(ls -td "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]* | head -1) || true
```

Comment:
This isn't doing what you think it is.
```
$ mkdir  kube-apiserver-pod-{99,100,101}
$ ls -td kube-apiserver-pod-[0-9]* | head -1
kube-apiserver-pod-100
```

Reply:
-t sorts by time. So, if 101 is the last modified directory, it should get it.

Reply:
gotcha ok `ls` can handle versions internally `ls -v | tail -1` but if time unique per revision we should be good. Do we have any other corner cases to consider?